upi/src/main/java/com/adyen/checkout/upi/internal/ui/view/UpiView.kt

```diff
@@ -87,6 +87,7 @@ internal class UpiView @JvmOverloads constructor(
     }
 
     private fun initPicker(delegate: UpiDelegate) {
+        binding.toggleButtonChoice.check(R.id.button_vpa)
```

Comment:
Does this trigger the `OnButtonCheckedListener`?

Reply:
It does, but the listener is added so it's okay